scripts/push_docker.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -e
+source "$(dirname $(realpath $0))/common.sh"
+
+tag_and_push_docker_image() {
+  local arch=$1
+
+  docker tag $IMAGE-$arch $ECR_URL/$IMAGE-$arch
```

Comment:
If I read `common.sh` correctly `$IMAGE` already contains the architecture. So are we creating images of name `foo:bar-arch-arch`? Do we actually want to do that?

Reply:
`$(arch)' got removed from image to allow multi architecture build and tagging of images, `-arch-arch` type of situations should be avoided under all circumstances

Reply:
Either way, that change to the variables in common should probably be put together with the changes to the user of these variables.

nit: The title of commit 4 is masking that quite a few things are happening apart from formatting. Ideally I'd split the formatting part from any functional changes to make reasoning about changes easier.